src/rtm/data_structure.rs

```diff
@@ -0,0 +1,151 @@
+use std::fmt::{Display, Result, Formatter};
+
+use json::{JsonValue};
+use chrono::{UTC, DateTime};
+
+#[allow(dead_code)]
+pub struct Team {
+    id: Option<String>,
+    name: Option<String>,
+    subdomain: Option<String>,
+    email_domain: Option<String>,
+    description: Option<String>,
+    inactive: Option<bool>,
+    logo_url: Option<String>,
+//    created: DateTime<UTC>,
+//    updated: DateTime<UTC>,
+}
+
+impl Display for Team {
+    fn fmt(&self, f : &mut Formatter) -> Result {
+        write!(f, "<Team ");
+
+        match self.id.clone() {
+            Some(id) => { write!(f, "id: \"{}\"", id); },
+            None => {}
+        }
+
+        match self.name.clone() {
+            Some(name) => { write!(f, ", name: \"{}\"", name); },
+            None => {}
+        }
+
+        write!(f, ">")
+    }
+}
+
+impl Team {
+    pub fn parse_from_json(mut data: JsonValue) -> Team {
```

Comment:
Why not use the [autoserialization](https://docs.serde.rs/rustc_serialize/json/index.html#using-autoserialization) ?